### PR TITLE
DOC: Add further aspects of dMRI to discussion

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add further aspects of dMRI to discussion.

### CODE REVIEW

File: _extras/discuss.md [hunk 1]
```diff
@@ -1,6 +1,40 @@
...
 title: Discussion
 ---
-FIXME
+
+## Multi-shell dMRI data
+
+The lesson presented a way to recover the fODFs on single shell data using the
+Constrained Spherical Deconvolution method. This is also called Single-Shell
+Single-Tissue (SSST) CSD since only the white matter tissue is used in the
+process. Having multiple shell data allows to decompose the contribution of
```
Comment (kaitj) [inline _extras/discuss.md:10]: Would slightly rephrase to "allows for decomposition of the contribution of different tissues"

File: _extras/discuss.md [hunk 2]
```diff
@@ -1,6 +1,40 @@
...
+different tissues (as different macroscopic tissues have unique b-value
+dependencies) to the fODF using the so-called Multi-Shell Multi-Tissue CSD
+(MSMT-CSD) method. Performing a multi-tissue decomposition using single
+shell data (Single-Shell 3-Tissue CSD, SS3T-CSD) has also been proposed.
```
Comment (kaitj) [inline _extras/discuss.md:14]: Wondering if references here would be helpful for MSMT and SS3T?

File: _extras/discuss.md [hunk 3]
```diff
@@ -1,6 +1,40 @@
...
+structural wiring of the brain allows to study its evolution and do comparative
+biology analyses to study the brain's similarities across species.
+
+# Limitations of dMRI
+
+The power of diffusion MRI lies in its ability to offer a non-invasive method
+to study the *in vivo* human brain. Although the resolution and the information
+made available from the dMRI data is improving over time (using more powerful
+MRI machines and other new acquisition sequences), it still does not allow to
+reach the level of detail that invasive method offer.
```
Comment (kaitj) [inline _extras/discuss.md:37]: invasive methods

**Review by kaitj — COMMENTED** [review]
Looks good, just a couple of changes to phrasing / spelling.

Also think that maybe some references could be helpful. Could just say they are outside the scope of these lessons, but here are references if interested?

File: _extras/discuss.md (hunk 1 above)
Comment (jhlegarreta) [reply _extras/discuss.md]: Done in cae0647.

File: _extras/discuss.md (hunk 2 above)
Comment (jhlegarreta) [reply _extras/discuss.md]: Done in cae0647.

File: _extras/discuss.md (hunk 3 above)
Comment (jhlegarreta) [reply _extras/discuss.md]: Done in cae0647.

**Comment by jhlegarreta** [comment]
> Also think that maybe some references could be helpful. Could just say they are outside the scope of these lessons, but here are references if interested?

Addressed in cae0647. I have not mentioned that they are out of the scope of the lesson; IMHO the fact that we added the discussion highlights the relevance. I think the provided information is enough so that learners are aware of it.

**Review by kaitj — APPROVED** [review]
> I have not mentioned that they are out of the scope of the lesson; IMHO the fact that we added the discussion highlights the relevance. I think the provided information is enough so that learners are aware of it.

That is fair - I do agree that it is relevant, and important especially if someone going through this is planning on applying tractography for their research. By "out of scope", I meant they should probably read the paper for details as outside of making them aware these methods exist, we probably wouldn't going go into detail about this (at this time anyway). Probably not the correct choice of phrasing from my last message. 

Nonetheless, sounds & looks good to me! Happy to merge once @josephmje has a chance to look / comment on the PR.

**Comment by jhlegarreta** [comment]
:+1: There references were added, so I think that everything is in place.

File: _extras/discuss.md [hunk 4]
```diff
@@ -24,6 +24,41 @@ The mentioned concepts are common across different 3D visualization packages.
...
+using dMRI data and tractography. Tractography allows to obtain many more
+derivatives, such as the groups of coherent streamlines (bundles) using
+*bundling* methods or different measures or scalar maps along bundles
+(*tractometry*) using the scalars, such as FA, computed from dMRI data. When
+combined with structural and functional data, diffusion MRI derivatives can be
+used to estimate the brain *connectivity* across different brain regions (using
+structural MRI parcellations). This is useful, among others, to study diseases
+(neurological or not) affecting the brain or different aspects related to its
+functional patterns (such as attention, learning, etc.). Similarly, studying the
+structural wiring of the brain allows to study its evolution and do comparative
```
Comment (josephmje) [inline _extras/discuss.md:51]:
```suggestion
structural wiring of the brain allows one to study its evolution and do comparative
```

Comment (josephmje) [inline _extras/discuss.md:51]:
```suggestion
structural wiring of the brain allows us to study its evolution and do comparative
```

File: _extras/discuss.md [hunk 5]
```diff
@@ -24,6 +24,41 @@ The mentioned concepts are common across different 3D visualization packages.
...
+biology analyses to study the brain's similarities across species.
+
+# Limitations of dMRI
+
+The power of diffusion MRI lies in its ability to offer a non-invasive method
+to study the *in vivo* human brain. Although the resolution and the information
+made available from the dMRI data is improving over time (using more powerful
+MRI machines and other new acquisition sequences), it still does not allow to
```
Comment (josephmje) [inline _extras/discuss.md:59]:
```suggestion
MRI machines and other new acquisition sequences), it still does not allow us to
```

File: _extras/discuss.md [hunk 6]
```diff
@@ -24,6 +24,41 @@ The mentioned concepts are common across different 3D visualization packages.
...
+reach the level of detail that invasive methods offer.
```
Comment (josephmje) [inline _extras/discuss.md:60]:
```suggestion
reach the level of detail that invasive methods, such as chemical tracers or anatomical sectioning, offer.
```

File: _extras/discuss.md (hunk 4 above)
Comment (jhlegarreta) [reply _extras/discuss.md]: Done in fd0e235.

File: _extras/discuss.md (hunk 5 above)
Comment (jhlegarreta) [reply _extras/discuss.md]: Done in fd0e235.

File: _extras/discuss.md (hunk 6 above)
Comment (jhlegarreta) [reply _extras/discuss.md]: Done in fd0e235.

**Review by josephmje — APPROVED** [review]

**Pull request merged**